packages/date-parser/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@holistics/date-parser",
-  "version": "3.1.2",
+  "version": "3.2.0",
```

Comment:
There's a (minor?) concern when integrating this back into Holistic. I'm not sure if the `yyyy-mm` syntax is accidentally used in Holistics, and the user somehow accepts this below behavior. And now we change the behavior to cover the whole month

![image](https://github.com/holistics/js/assets/11194590/88ff46b0-394f-4c45-b88f-eb9211714a32)

Reply:
Minor: because the above behavior looks weird, so maybe nobody is using it

Reply:
I think it's safe because `2023-01` is currently parsed as a time of the current date, so the behavior is too uncanny to use.

Reply:
it recognizes that input as `from 20:23 to 01:00` 
![image](https://github.com/holistics/js/assets/15865062/7414f8cf-6a07-4cde-bf1c-becacde193c5)

Shouldn't be an issue because:
* I don't think it is really usable
* We do not document it https://docs.holistics.io/docs/datetimes/relative-dates -> We don't say it's supported

Reply:
@khanhhuy  @datcv-holistics FYI